app/shared/models.py

```diff
@@ -2,10 +2,10 @@
 
 from django.db import models
 
+
 class CommonInfo(models.Model):
```

Comment:
Hey @andela-uawili breaking the `CommonInfo` model into a set of reusable mixin seems cool though, but a couple of questions:

1) Generally, composition is favored over multiple inheritance  which makes me wonder how many behavioural mixins we are going to have (if we are going to have a lot of behavioral mixins then having our models inheriting them all may not be the best thing to do)

2) My idea of the `CommonInfo` model was just to house the common fields and every other model inheriting it implements their own needed fields. If this is the case, do we still need to break it up? How large could the `CommonInfo` field become?

3) Maybe we should look at our implementation of the `CommonInfo` model now, so far we only have the time fields, would every model inheriting from the `CommonInfo` model need those two fields ? Maybe we should be focusing more on the Interface segregation principle and the Dependency Inversion principle. Trying to avoid a YAGNI (You aren't gonna need it) situation.

cc @andela-kanyanwu